requests/cluster/cluster.go

```diff
@@ -5,19 +5,10 @@ import (
 	"fmt"
 	"net/http"
 
-	l "github.com/redhatinsights/uhc-auth-proxy/logger"
 	"github.com/redhatinsights/uhc-auth-proxy/requests/client"
 	"github.com/spf13/viper"
-	"go.uber.org/zap"
 )
 
-var log *zap.Logger
-
-func init() {
-	l.InitLogger()
-	log = l.Log.Named("cluster")
```

Comment:
I think it might be better to add some logs instead of removing the logger if its not used. Maybe we can log the errors returned in the methods below? What do you think?